install.sh

```diff
@@ -269,9 +262,9 @@ fi
 # Set the required OS based on inputs and installed distribution
 if [[ "$INSTALL_NAV" == "true" ]]; then
```

Comment:
Get rid of this whole special check for INSTALL_NAV. All projects will use 16.04 or debian now

Reply:
Kept part of it as it is used to set quite a few variables later down the line, keeping the check as install nav just works there. If preferred I can remove that and have the variables declared somewhere else or inline but it is rather neat and concise as it is now.